ansible/roles/packer-cleanup/tasks/main.yml

```diff
@@ -15,3 +15,24 @@
   file:
     dest: /etc/machine-id
     state: touch
+
+- name: Clean up extraneous /etc/hosts entries
```

Comment:
@vincepri @chuckha do you think this is something that would be valuable in the capa image baking as well?

Reply:
I'm not entirely sure why these clean up rules exist -- were these causing some problem? I'm guessing they did or this PR wouldn't exist.

@scottslowe what was the issue that led to these changes? This will help us know if we need to port this over to CAPA.

Reply:
@chuckha We observed extraneous `/etc/hosts` entries when building OCI (Oracle Cloud) images. I haven't observed similar behaviors for AWS images, so this probably does not need to be ported for CAPA (IMO).

Reply:
Perfect, thanks so much for the info :D

Reply:
Out of curiosity though, why were the extraneous /etc/hosts a problem when building oracle cloud images?

Reply:
Initially, we believed the extraneous entries to be contributing to some issues with `kube-proxy`. After the PR was submitted, we determined that Oracle had hardware issues with some of their cloud hardware that dropped all encapsulated traffic (like Calico's IP-in-IP traffic). In theory, that meant the changes in the PR weren't absolutely necessary. We kept the PR open because (to be honest) we wanted to image builds to be as "clean" as possible.